pkg/service/v0/accounts.go

```diff
@@ -81,14 +83,6 @@ func (s Service) loadAccount(id string, a *proto.Account) (err error) {
 	return
 }
 
-// loggableAccount redacts the password from the account
-func loggableAccount(a *proto.Account) *proto.Account {
```

Comment:
@butonic Replaced this with https://github.com/owncloud/ocis-accounts/pull/69/files#diff-1bc5586c5f4afccf828e3161c29aa9f2R534 as this function was modifying the reference thus causing the json-store to having the "****REMOVED***" string